js/index.js

```diff
@@ -29,11 +21,69 @@ document.addEventListener("DOMContentLoaded", () => {
     "Project Management Systems (Jira, Asana, Wrike)",
     "Email Marketing (Klaviyo, Mailchimp)",
   ];
-  const skillsSection = document.getElementById('skills');
-  skillsList = skillsSection.querySelector('ul');
+  const skillsSection = document.getElementById("skills");
+  skillsList = skillsSection.querySelector("ul");
   for (let i = 0; i < skills.length; i++) {
     const skill = document.createElement("li");
     skill.innerHTML = skills[i];
     skillsList.appendChild(skill);
   }
+
+
+
+  /////////////////////////////////////////
+  ///////////// MESSAGE FORM /////////////
+  ///////////////////////////////////////
+
+  // HIDE MESSAGES WHEN NONE//
+  document.getElementById("messages").style.display = "none";
+
+  // FORM //
+  const messageForm = document.getElementsByName("leave_message")[0];
+  messageForm.addEventListener("submit", (e) => {
+    e.preventDefault();
+
+    // USER INFO //
+    const usersName = e.target.usersName.value;
+    const usersEmail = e.target.usersEmail.value;
+    const usersMessage = e.target.usersMessage.value;
+    const newMessage = document.createElement("li");
+    newMessage.innerHTML = `<a href="mailto:${usersEmail}">${usersName} </a> wrote:<span> ${usersMessage}</span>`;
```

Comment:
You mentioned: 
> I wasn't able to quite figure out the edit button stuff for extra credit but will try more this week.

There are different ways to do this. You could try https://developer.mozilla.org/en-US/docs/Web/HTML/Global_attributes/contenteditable, but if you're taking a different approach, that can be good too.